js/utils.js

```diff
@@ -111,12 +112,53 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
   }
 
   function registerHandlebarsHelpers() {
-    Handlebars.registerHelper('formatDate', function(date) {
-      if (!date) {
-        return;
+    Handlebars.registerHelper('plaintext', function(context) {
```

Comment:
`plaintext` and `removeSpaces` should not be added. They were removed in https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/534 because they are now declared in our core libraries.